docs/source/user/debugger.rst

```diff
@@ -16,29 +16,18 @@ Here is a list of kernels that are known to be supporting the Jupyter Debug Prot
 
 - `xeus-python <https://github.com/jupyter-xeus/xeus-python>`_: Jupyter kernel for the Python programming language
 - `xeus-robot <https://github.com/jupyter-xeus/xeus-robot>`_: Jupyter kernel for Robot Framework
+- `ipykernel <https://github.com/ipython/ipykernel>`_:  IPython Kernel for Jupyter 
+
 
 Other Jupyter Kernels can also support debugging and be compatible with the JupyterLab debugger
 by implementing the `Jupyter Debugger Protocol <https://jupyter-client.readthedocs.io/en/latest/messaging.html#debug-request>`_.
 
 If you know of other kernels with support for debugging, please open a PR to add them to this list.
 
-Here is an example of how to install ``xeus-python`` in a new ``conda`` environment:
-
-.. code:: bash
-
-   conda create -n jupyterlab-debugger -c conda-forge jupyterlab=3 xeus-python
-   conda activate jupyterlab-debugger
-
 Usage
 -----
 
-``xeus-python`` can be selected from the JupyterLab launcher:
-
-.. image:: ./images/debugger/xpython-launcher.png
-
-Alternatively, it is also possible to switch to the `xpython` kernel using the kernel selection dialog:
-
-.. image:: ./images/debugger/kernel-selection.gif
+For Python, both ``ipykernel`` and ``xeus-python`` support debugging.
```

Comment:
Would it be a good idea to add the minimum required version of `ipykernel` in a parentheses? I am afraid that otherwise users with older versions will be confused seeing that `ipykernel` they have installed does not seem to activate debugger.

Reply:
Good point :+1: 

Maybe we can actually add the conda environment creation back to the docs to specify `ipykernel=6`

Reply:
Done in https://github.com/jupyterlab/jupyterlab/pull/10512/commits/9255e8111f1e29f36b9e46645a2b2f259229ce5a